README.rst

```diff
@@ -39,6 +39,8 @@ How to use
     ['Dev', 'Living Room', 'Den', 'Bedroom']
 
     >> cast = next(cc for cc in chromecasts if cc.device.friendly_name == "Living Room")
+    >> # Start worker thread
+    >> cast.start()
```

Comment:
Thinking here, do you think that we can automatically call `start` inside `wait`, if not started yet ? That way it's a less breaking change.